examples/prefect/benchmark.py

```diff
@@ -0,0 +1,19 @@
+import numpy as np
+from prefect import task
+from sklearn import base, metrics
+
+import nnbench
+
+
+@nnbench.benchmark
```

Comment:
I would leave this file out until we can think of a way to facilitate the nnbench.Benchmark<->prefect.Task correspondence.

Reply:
Damn, yeah. I deleted it but it must have somehow come back  while squashing when i was cleaning up the git history.